sdk/confidentialledger/azure-security-confidentialledger/test-resources.json

```diff
@@ -2,56 +2,53 @@
 	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
 	"contentVersion": "1.0.0.0",
 	"parameters": {
-		"ledgerName": {
+		"BaseName": {
 			"type": "string",
 			"metadata": {
 				"description": "Ledger Name"
-			},
-			"minLength": 3,
-			"maxLength": 24
+			}
 		},
-		"principalId": {
+		"TestApplicationId": {
 			"type": "string",
 			"metadata": {
 				"description": "Oid of the user"
 			}
 		},
 		"location": {
 			"type": "string",
-			"defaultValue": "[resourceGroup().location]",
+			"defaultValue": "eastus",
```

Comment:
Is there a specific reason we are hard-coding the region?

Reply:
`ResourceGroup().location` returns `westus` and this isn't supported yet by the service.